auto_submit/test/server_test.dart

```diff
@@ -0,0 +1,21 @@
+// Copyright 2021 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:shelf/shelf.dart';
+import 'package:test/test.dart';
+import 'package:auto_submit/requests/github_webhook.dart';
+
+void main() {
+  Request req = Request('POST', Uri.parse('http://localhost/'),
+      headers: {
+        'header1': 'header value1',
+      },
+      body: "{\"label1\": \"label1cotent\"}");
+  GithubWebhook githubWebhook = GithubWebhook();
```

Comment:
Avoid adding logic directly in main. It makes it harder to debug and find what code is being run. Adding basic data models is fine

This should either be in test(...); or in a `setUp(...);`